### ISSUE TITLE
yallo.tv detection & ads

### ISSUE THREAD

**Issue by etwasch** [uBlockOrigin/uAssets#19227]
@mapx- 
> Does this help ?
> `@@||api.production.k8s.y3o.tv/*/video-ads$xhr,domain=yallo.tv`

Since a few days this doesn't work anymore.
Here is the new log, can you help again?
<img width="1920" alt="Bildschirmfoto 2023-07-30 um 00 02 12" src="https://github.com/uBlockOrigin/uAssets/assets/15729562/7f3f83f4-e393-484c-ab0f-a892640b5e1c">

_Originally posted by @etwasch in https://github.com/uBlockOrigin/uAssets/issues/16014#issuecomment-1656927732_

**Comment by stephenhawk8054** [comment]
There's a new update already: https://github.com/uBlockOrigin/uAssets/commit/f97be18c48b2a7d067652df7b781651d932e231a

Does it work for you?

**Comment by etwasch** [comment]
> There's a new update already: [f97be18](https://github.com/uBlockOrigin/uAssets/commit/f97be18c48b2a7d067652df7b781651d932e231a)
> Does it work for you?

No, I think that commit just removed the unused and non-working rules.
I updated the screenshot of the uBlock protocol log in the post above.

**Comment by stephenhawk8054** [comment]
Then please just continue in the old thread.

**Comment by etwasch** [comment]
Ok, thank you for reopening it.
